pkg/registry/bundlegraphloader_test.go

```diff
@@ -2,6 +2,7 @@ package registry
 
 import (
 	"encoding/json"
+	"github.com/blang/semver"
```

Comment:
nit: import order

remember, even with goimports, we still need to organize the imports into the correct groups; e.g.
```
import (
    <standard>
    
    <external orgs>

    <same org>
)

Reply:
It'd be nice if go fmt did this for free